src/Microsoft.Azure.ServiceBus/Primitives/ServiceBusConnection.cs

```diff
@@ -66,23 +66,16 @@ static void CloseConnection(AmqpConnection connection)
         async Task<AmqpConnection> CreateConnectionAsync(TimeSpan timeout)
         {
             string hostName = this.Endpoint.Host;
-            string networkHost = this.Endpoint.Host;
-            int port = this.Endpoint.Port;
 
             TimeoutHelper timeoutHelper = new TimeoutHelper(timeout);
             AmqpSettings amqpSettings = AmqpConnectionHelper.CreateAmqpSettings(
```

Comment:
>amqpSettings [](start = 25, length = 12)

While you are at it, could you please pass `true` for `useWebSockets` in this method in the case of web-sockets